CRM/Report/Form/Contribute/Summary.php

```diff
@@ -229,14 +229,24 @@ public function __construct() {
       'civicrm_batch' => array(
         'dao' => 'CRM_Batch_DAO_EntityBatch',
         'grouping' => 'contri-fields',
-        'filters' => array(
+        'fields' => array(
           'batch_id' => array(
+            'name' => 'batch_id',
+            'title' => ts('Batch Title'),
+          ),
+        ),
+        'filters' => array(
+          'bid' => array(
```

Comment:
Just a question - why have you called this 'bid' rather than 'batch_id' - it seems confusing (& would unset it on existing reports) and I can't quote see what it achieves.